document-portal/xdp-main.c

```diff
@@ -47,6 +47,7 @@ static GQueue get_mount_point_invocations = G_QUEUE_INIT;
 static XdpDbusDocuments *dbus_api;
 FlatpakDir *user_dir = NULL;
 FlatpakDir *system_dir = NULL;
+GHashTable *app_runtimes = NULL;
```

Comment:
This is not really needed but I do not like the idea of parsing the metadata key file each time we need to resolve a path.
BTW is there a better way to get the runtime path from an app id?